package/gargoyle/files/www/login.sh

```diff
@@ -70,13 +70,16 @@ var passInvalid = false;
 <div id="login_status" class="alert alert-danger" role="alert" style="display:none;"></div>
 
 <div class="row">
-	<div class="col-lg-12">
-		<div class="panel panel-primary">
+	<div class="col-lg-8">
+		<div class="panel panel-default">
+			<div class="panel-heading">
+				<h3 class="panel-title"><%~ EAdmP %></h3>
+			</div>
 			<div class="panel-body">
 				<div class="row form-group">
 					<label class="sr-only" for="password" id="password_label"><%~ EAdmP %></label>
 					<span class="col-xs-12">
-						<input id="password" class="form-control" type="password" onkeyup="proofreadLengthRange(this,1,999)" onkeydown="checkKey(event)" size="25" placeholder="<%~ EAdmP %>"/>
+						<input id="password" class="form-control" type="password" onkeyup="proofreadLengthRange(this,1,999)" onkeydown="checkKey(event)" size="25"/>
```

Comment:
Why is placeholder removed? I don't agree with this.

Reply:
Thought it would be redundant. No problem, I'll add it back.

Should I remove the newly added panel-heading/title as well?

Reply:
Sorry i was only looking very localised, didn't see you had replaced it with panel title.
I think i liked it better how it was originally, but it's a personal taste thing and not something i'm attached to.

If you think this is functionally better, put it in :+1:

Reply:
No worries, I'm adding it back. I'll just leave the changes related to theme panel style not getting properly applied.